pkg/plugins/builtin/http/http.go

```diff
@@ -57,9 +58,10 @@ type parameter struct {
 
 // auth represents HTTP authentication
 type auth struct {
-	Basic     *authBasic `json:"basic"`
-	Bearer    *string    `json:"bearer"`
-	MutualTLS *mTLS      `json:"mutual_tls"`
+	Basic     *authBasic  `json:"basic"`
+	Digest    *authDigest `json:"digest"`
```

Comment:
This new authentication would need to be documented inside the plugin README: https://github.com/ovh/utask/blob/master/pkg/plugins/builtin/http/README.md